src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1962,6 +1962,16 @@ elementFormDefault="qualified">
             <xs:documentation><!-- _locID_text="OutputType" _locComment="" -->Type of output to generate (WinExe, Exe, or Library)</xs:documentation>
         </xs:annotation>
     </xs:element>
+    <xs:element name="UseArtifactsOutput" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="UseArtifactsOutput" _locComment="" -->Indicate whether to opt into the centralized output path format. All build outputs from all projects are gathered into a common location, separated by project.</xs:documentation>
```

Comment:
Are links possible in these definitions? If so I'd like to point to the documentation we have for the Artifacts layout at https://learn.microsoft.com/en-us/dotnet/core/sdk/artifacts-output.

Reply:
Added into the annotation which is displayed in the tooltip.